Generate.py

```diff
@@ -14,44 +14,42 @@
 
 ModuleUpdate.update()
 
+import copy
 import Utils
+import Options
+from BaseClasses import seeddigits, get_seed, PlandoOptions
+from Main import main as ERmain
+from settings import get_settings
+from Utils import parse_yamls, version_tuple, __version__, tuplize_version, user_path
 from worlds.alttp import Options as LttPOptions
-from worlds.generic import PlandoConnection
-from Utils import parse_yamls, version_tuple, __version__, tuplize_version, get_options, user_path
 from worlds.alttp.EntranceRandomizer import parse_arguments
-from Main import main as ERmain
-from BaseClasses import seeddigits, get_seed, PlandoOptions
-import Options
 from worlds.alttp.Text import TextTable
 from worlds.AutoWorld import AutoWorldRegister
-import copy
+from worlds.generic import PlandoConnection
 
 
 def mystery_argparse():
-    options = get_options()
-    defaults = options["generator"]
-
-    def resolve_path(path: str, resolver: Callable[[str], str]) -> str:
-        return path if os.path.isabs(path) else resolver(path)
+    options = get_settings()
```

Comment:
The idea is to name the new thing as settings to differentiate between world *settings* and player *options*. However as long as the old API (`Util.get_options()`) is still around, it's gonna be confusing.

Reply:
Oh, to clarify: some things inside the settings are currently called "options" even though "settings" would be better. The reason for this is to maintain compatibility to existing host.yamls. In a future update we might be able to migrate this, but that requires the worlds to use the settings API rather than Utils.get_options()

As for the specific line you marked, I tried to touch existing code as little as possible, so local variables have the same (old) name for the most part. "Fixing" that and using the settings API throughout core will be a future endeavour.
As for uses of "options" in `settings.py`, all of them are deliberate and some or most of them will hopefully be renamed in the future.